example/example18.cpp

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
That is how it is supposed to be. Just add a test() function that becomes part of the library, and which you call from python (and which in turn calls back to Python via eval()).